src/test/kotlin/io/openfuture/state/base/RedisRepositoryTests.kt

```diff
@@ -10,8 +10,6 @@ import org.springframework.data.redis.core.ReactiveRedisTemplate
 @DataRedisTest
 @Import(RedisConfig::class, JacksonAutoConfiguration::class)
 abstract class RedisRepositoryTests {
-
     @Autowired
-    protected lateinit var redisTemplate: ReactiveRedisTemplate<String, Any>
-
+    protected lateinit var reactiveRedisTemplate: ReactiveRedisTemplate<String, Any>
```

Comment:
`reactiveRedisTemplate` -> `redisTemplate`

You don't have any redisTemplate that is not reactive, right? No reason to specify it at the name of the variable.